validator/client/attest_protect_test.go

```diff
@@ -54,6 +54,26 @@ func TestPreSignatureValidation(t *testing.T) {
 	mockProtector.AllowAttestation = true
 	err = validator.preAttSignValidations(context.Background(), att, pubKey)
 	require.NoError(t, err, "Expected allowed attestation not to throw error")
+
+	e, exists, err := validator.db.LowestSignedSourceEpoch(context.Background(), pubKey)
+	require.NoError(t, err)
+	require.Equal(t, true, exists)
+	require.Equal(t, uint64(0), e)
+	e, exists, err = validator.db.LowestSignedTargetEpoch(context.Background(), pubKey)
+	require.NoError(t, err)
+	require.Equal(t, true, exists)
+	require.Equal(t, uint64(0), e)
+
+	m.validatorClient.EXPECT().DomainData(
+		gomock.Any(), // ctx
+		gomock.Any(), // epoch
```

Comment:
Can you specify the value here? What do you expect to be sent? It looks like 0, which is a default value. Also consider a non-default value to make this test more robust.